tools/chrome-debug-extension/README.md

```diff
@@ -0,0 +1,75 @@
+# Application Insights Debug Viewer
+
+## What the tool does
+
+This tool runs in your browser, either Chrome or Edge, and provides you real time visualization of events flowing through Application Insights. You can use it to monitor the telemetry that your web application is emitting as part of your dev inner loop or bug investigations, or you can use it to monitor the internal calls within Application Insights to debug your integration of Application Insights into your web application.
+
+We recommend building a configuration file to represent the schema of the telemetry your web application emits, and have your engineering team use it during the dev inner loop to ensure their changes are emitting the expected telemetry, and to also catch warnings/errors early.
+
+## Installing the tool
+
+1. Download either the [official](https://js.monitor.azure.com/release/tools/ai.chrome-ext.zip) or [nightly](https://js.monitor.azure.com/nightly/tools/ai.chrome-ext.zip) build and unzip it into a folder on your PR
+1. Open the Manage Extensions page in either Chrome or Edge
+   - Browse to either edge://extensions or chrome://extensions/ respectively
+   - Or choose Extensions from the ... menu
+1. Enable Developer Mode
+1. Click `Load unpacked`
+1. Browse to the path where you unzipped the extension
+1. Click `Select Folder`
+
+## Using the tool
+
+1. To launch the tool, click the ![Telemetry-Viewer-icon](images/icon-19.png) icon next to the address bar
+    - If it isn't there, click the `small puzzle piece` icon and then the `Telemetry Viewer`![Telemetry-Viewer-icon](images/icon-19.png)
+1. The first time you run the tool, it will start in the `Configuration Selection` page, which you can always return to by clicking the gear icon later
+    - If your team has a preset configuration in the drop down list, select it
+    - Otherwise, use you can start with the Default configuration
```

Comment:
This sentence is not very clear to me.

Reply:
Are you referring to "Otherwise, use you can start with the Default configuration"?
The configuration selected when they first open the tool will be named Default:
![image](https://user-images.githubusercontent.com/51932180/142479392-c8931fcd-1fd3-4969-b24e-b025ba8a6622.png)

How would you recommend clarifying this?  Thanks!